docs/src/pages/guides/examples.md

```diff
@@ -70,11 +70,11 @@ probes:
           password: somepassword
 ```
 
-Using the configuration above, Monika will send a POST request to `http://www.foo.com/login.php` with the defined request's body.
+Using the configuration above, Monika will send a POST request to `http://www.example.com/login.php` with the defined request's body.
 
-## Multiple request
+## Multiple requests
 
-Here is an example configuration with multiple requests:
+Monika support multiple requests, below is an example configuration:
```

Comment:
Monika supports sending multiple requests **one after another** in a single probe.